peer/core.yaml

```diff
@@ -335,6 +335,13 @@ ledger:
       # Deploy chaincodes into the genesis block
       chaincodes:
 
+        noop_syscc:
+          path: github.com/hyperledger/fabric/core/system_chaincode/noop
+          type: GOLANG
+          constructor:
+            args:
+
+
```

Comment:
@muralisrini this chain code does not mean to decrypt, it is plain access to data as is on the block chain.